CMakeLists.txt

```diff
@@ -703,8 +703,8 @@ target_include_directories(
 
 install(
   DIRECTORY
-  ${CUTLASS_INCLUDE_DIR}/
-  ${CMAKE_CURRENT_BINARY_DIR}/include/
+  ${CUTLASS_INCLUDE_DIR}
+  ${CMAKE_CURRENT_BINARY_DIR}/include
```

Comment:
The `${CMAKE_CURRENT_BINARY_DIR}/include` looks fishy to me. I'm trying to recall if the trailing `/` was used to get the contents of the include folder copied into the `DESTINATION` folder. Can you show the install diffs to see what moved?

Reply:
Yeah, removing these trailing `/`s is causing the entire folder to be copied to the given `DESTINATION` instead of the contents. These two lines need to be reverted.